<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1188</width>
    <height>766</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1171</width>
      <height>711</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Snap ITC</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
height: 50px;
background-color: black;
  color: #fff;
  font-size: 30px;
  border-radius: 0px;
  padding: 5px 15px;
  margin-top: 10px;
  outline: 0px;
}</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="game">
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1171</width>
        <height>711</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="menu">
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>60</y>
        <width>361</width>
        <height>571</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="lvl_1">
         <property name="font">
          <font>
           <family>Sitka Text</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Level 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="lvl_2">
         <property name="font">
          <font>
           <family>Sitka Text</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Level 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="lvl_3">
         <property name="font">
          <font>
           <family>Sitka Text</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Level 3</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="settings_b">
         <property name="font">
          <font>
           <family>Sitka Text</family>
           <pointsize>-1</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Settings</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="settings">
     <widget class="QCheckBox" name="checkBox">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>160</y>
        <width>201</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Text</family>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Music</string>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>260</y>
        <width>461</width>
        <height>231</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <property name="horizontalSpacing">
        <number>20</number>
       </property>
       <property name="verticalSpacing">
        <number>35</number>
       </property>
       <item row="0" column="1">
        <widget class="QKeySequenceEdit" name="keyUp">
         <property name="font">
          <font>
           <family>Sitka Text</family>
           <pointsize>22</pointsize>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QKeySequenceEdit" name="keyLeft">
         <property name="font">
          <font>
           <family>Sitka Text</family>
           <pointsize>22</pointsize>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QKeySequenceEdit" name="keyRight">
         <property name="font">
          <font>
           <family>Sitka Text</family>
           <pointsize>22</pointsize>
           <bold>true</bold>
          </font>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Sitka Text</family>
           <pointsize>22</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Key Up</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Sitka Text</family>
           <pointsize>22</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Key Left</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Sitka Text</family>
           <pointsize>22</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Key Right</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="save_b">
      <property name="geometry">
       <rect>
        <x>990</x>
        <y>520</y>
        <width>161</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Snap ITC</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="win_page">
     <widget class="QPushButton" name="to_menu_2">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>580</y>
        <width>341</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Text</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Go to main menu</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>170</y>
        <width>521</width>
        <height>131</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Text</family>
        <pointsize>40</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Congratulations!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>220</y>
        <width>521</width>
        <height>331</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Text</family>
        <pointsize>22</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="margin">
       <number>20</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="lose_page">
     <widget class="QPushButton" name="to_menu_1">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>580</y>
        <width>341</width>
        <height>91</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Text</family>
        <pointsize>-1</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Go to main menu</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>200</y>
        <width>431</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Text</family>
        <pointsize>40</pointsize>
       </font>
      </property>
      <property name="text">
       <string>You lose!</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1188</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
